completions/ssh.completion.sh

```diff
@@ -45,7 +45,7 @@ function _omb_completion_ssh {
     fi
   done
   if ((${#config_files[@]} != 0)); then
-    COMPREPLY+=($(compgen -W "$(awk '/^Host/ {for (i=2; i<=NF; i++) print $i}' "${config_files[@]}")" "${options[@]}"))
+    COMPREPLY+=($(compgen -W "$(awk '/^[Hh][Oo][Ss][Tt] /{for (i=2; i<=NF; i++) print $i}' "${config_files[@]}")" "${options[@]}"))
```

Comment:
You may also check [the regular expression used in `bash-completion`](https://github.com/scop/bash-completion/blob/8a8880db78e9b04fb3d66681028af701701f93f2/bash_completion#L2743-L2746).

By the way, the original regular expression `^Host` also matched `HostName`, which is followed by the full names of the hosts (in contrast to `Host` where the user can give a short name). This change would disable showing the full name of the hostnames.

Reply:
Ah. Didn't take into account that reading 'HostName' would be a good idea.
Have now changed the code to also read HostName in different cases. But without hitting other entries, like HashKnownHosts, HostbasedAcceptedAlgorithms, HostbasedAuthentication or HostKeyAlgorithms like it did before.

Reply:
Thanks! Yeah, right. It's even better to check `HostName` explicitly to avoid hitting other options.